cogip/tools/planner/avoidance/process.py

```diff
@@ -120,7 +120,13 @@ def avoidance_process(
         else:
             shared_properties["last_avoidance_pose_current"] = (pose_current.x, pose_current.y)
 
-            path = avoidance.get_path(pose_current, pose_order, dyn_obstacles, avoidance_strategy)
+            if pose_current.x == pose_order.x and pose_current.y == pose_order.y and pose_current.O != pose_order.O:
```

Comment:
I'm not confident about this. It should work in simulation because everything is perfect. But in reality, we'll be within a motion control stop window. So it won't be == but rather plus or minus X mm depending on the threshold.

Reply:
It should not cause any problem here because it is a situation where pose_order is computed from pose_current.